docs/ibexa_engage/integrate_ibexa_connect.md

```diff
@@ -0,0 +1,57 @@
+---
+description: Integrate Ibexa Engage with Ibexa Connect
+edition: experience
+---
+
+# Use Ibexa Connect
+
+You can create workflows using [Ibexa Connect](https://www.ibexa.co/products/features/integration-and-automation).
```

Comment:
Maybe we should link to our Connect guide here? It's something we control (I mean https://doc.ibexa.co/projects/connect/en/latest/general/ibexa_connect/ )